src/main/java/com/getindata/connectors/http/HttpSinkPostRequestCallback.java

```diff
@@ -0,0 +1,32 @@
+package com.getindata.connectors.http;
+
+import java.io.Serializable;
+import java.net.http.HttpResponse;
+import java.util.Map;
+
+import com.getindata.connectors.http.internal.sink.HttpSinkRequestEntry;
+import com.getindata.connectors.http.internal.table.sink.HttpDynamicSink;
+
+/**
+ * An interface for a custom class that logs responses and requests processed by the HTTP Sink.
```

Comment:
Could you change it so something like `An interface for post request callback action...`

So you would not mention anything about logs nor Sink/Source.

Reply:
done